src/core_plugins/timelion/public/webpackShims/jquery-flot.js

```diff
@@ -1,7 +1,16 @@
+const $ = require('jquery');
+
+if (window) window.jQuery = $;
```

Comment:
Ah, good to know. There's a bunch of places flot is being imported incorrectly then. I'll circle back to this PR in a bit. Thanks for the education @spalger!